modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2181,7 +2181,8 @@ public CompletableFuture<PartitionSet> localPartitionSetAsync(long causalityToke
         }
 
         try {
-            return localPartitionsVv.get(causalityToken).thenApply(unused -> localPartsByTableId.get(tableId));
+            return localPartitionsVv.get(causalityToken)
```

Comment:
Is using an empty set as a default value for VV an option? Have you considered it?

Reply:
`localPartitionsVv` is typed as `Void`. But I switched to `getOrDefault()` on the map.